cmd/influxd/config.go

```diff
@@ -455,3 +460,34 @@ func (g *Graphite) LastEnabled() bool {
 
 // maxInt is the largest integer representable by a word (architeture dependent).
 const maxInt = int64(^uint(0) >> 1)
+
+type OpenTSDB struct {
+	Addr string `toml:"address"`
+	Port int    `toml:"port"`
+
+	Enabled         bool   `toml:"enabled"`
+	Database        string `toml:"database"`
+	RetentionPolicy string `toml:"retention-policy"`
+}
+
+func (o OpenTSDB) DatabaseString() string {
+	if o.Database == "" {
+		return DefaultOpenTSDBDatabaseName
+	}
+	return o.Database
+}
+
+func (o OpenTSDB) ListenAddress() string {
```

Comment:
This function needs to take the default bind address, and use it if an address is not set for openTSDB. See https://github.com/influxdb/influxdb/blob/master/cmd/influxd/config.go#L390